pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -2583,6 +2583,19 @@ let
         };
       };
 
+      moshfeu.compare-folders = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "compare-folders";
+          publisher = "moshfeu";
+          version = "0.24.2";
+          sha256 = "sha256-EiGuYRMN8bXq+Cya38U+dCX2W0wzIeP0yb39WBJaX1U=";
+        };
+
+        meta = {
+          license = lib.licenses.mit;
```

Comment:
Please add missing meta information, just like pretty much all the other extensions in this file.